source/includes/_tile.md

```diff
@@ -20,6 +20,50 @@ This endpoint identifies which lock belongs to the specific tile.
 
```

Comment:
You should rename `## Get Lock Belonging To Tile` to `## Get Lock Belonging To Tile (v1)` and add a comment somewhere saying its deprecated in favour of v3